lib/rspec/controller_keyword_argument.rb

```diff
@@ -3,55 +3,64 @@
   actions = %w(get post put delete)
   within_files 'spec/controllers/**/**.rb' do
     with_node type: :block do
-      if node.caller.message == :it || node.caller.message == :expect
+      in_it_block =
+        begin
+          node.caller.message == :it || node.caller.message == :expect
+        rescue Synvert::Core::MethodNotSupported
+          false
+        end
```

Comment:
you can do like `with_node type: :block, caller: { type: 'send', message: %i[it expect] } do` to avoid this check

Reply:
Ok, I'll try that. thanks!